sources/packages/backend/apps/workers/src/controllers/cra-integration/cra-integration.dto.ts

```diff
@@ -0,0 +1,18 @@
+export class CreateIncomeRequestJobInDTO {
```

Comment:
do we need class for worker DTOs

Reply:
@ann-aot I don't think the workers will be exposed on swagger. So your question is valid.

Reply:
Nope, the ones from CRA were mistakenly created as classes. I will be converting to interfaces in the same way that others are.

Reply:
Changed to interfaces.